doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -1,223 +1,275 @@
+.When you morph to an event gateway, you can not delete the connected nodes and save the process [https://issues.redhat.com/browse/RHPAM-3036[RHPAM-3036]]
 
-Issue: In the process designer, the reusable sub process is not populated in the *Called element* drop-down list.
+Issue: When you morph to an event gateway, you can not delete the connected nodes and save the process. Also, it is not possible to delete the sequence flows. You receive an error message.
 
 Steps to reproduce:
 
-. Create process A.
-. Create process B.
-. Create reusable sub process in process B.
-. Click *Called element* property.
+* Create a process with a parallel gateway.
+* Append any node expect Timer and Text Annotation.
+* Morph that gateway to event gateway.
+* Try to delete any connected nodes, sequence flow or the gateway itself.
+* Save the process.
+
+Expected result: You can delete the connected nodes, sequence flow. You can save the process. Also, you receive a warning message in case the process is not executable.
+
+Actual result: You can not delete the connected nodes, sequence flow. Also, you can not save the process.
+
+Workaround: Change a gateway type back to its previous gateway type using morphing toolbox menu. For example, try to change the event gateway back to parallel gateway.
+endif::[]
+
+== Installer
 
-Expected result: Called element property is populated.
+.The {PRODUCT} installer contains references to {JWS} 5.2 [https://issues.redhat.com/browse/RHPAM-3077[RHPAM-3077]]
 
-Actual result: Called element property is not populated.
+Issue: When you run the {PRODUCT} installer, the installation path step references to {JWS} 5.2 instead of {JWS} {JWS_VERSION}.
+
+Workaround: None.
 
-Workaround: Deactivate the Reusable sub-process and activate it again.
+== DMN Designer
 
-.An error occurs when editing the process during restoring the other process [https://issues.redhat.com/browse/RHPAM-2757[RHPAM-2757]]
+.When you include a PMML 4.4 model into a DMN model and then save or validate the model, DMN throws an error [https://issues.redhat.com/browse/RHDM-1400[RHDM-1400]]
 
-Issue: In the process designer, While editing the process when you try to restore other process an error message is shown.
+Issue: You can not include a PMML 4.4 model into a DMN model, DMN model throws an error.
 
 Steps to reproduce:
 
-. Create process A and do not close this process.
-. Create process B.
-. Make a change in process B and click *Save*.
-. Click *Latest version* and select *Version 1*. Do not click *Restore*.
-. Open the process A.
-. Make a change in process A and click *Save*.
+* Create a new empty project in {CENTRAL}.
+* Create an empty DMN model.
+* Import one PMML asset with the 4.4 version into the project.
+* Open the DMN model.
+* Select the *Included Models* tab.
+* Click the *Include Model* and select the PMML 4.4 model.
+* Click *Include*.
 
```

Comment:
I'd suggest using a numbered list instead of bullets since these are sequential steps.